tests/integ/modin/frame/test_head_tail.py

```diff
@@ -71,3 +71,28 @@ def test_empty_dataframe(n, empty_snowpark_pandas_df):
         comparator=eval_result_and_query_with_no_join,
         check_column_type=False,
     )
+
+
+@pytest.mark.parametrize(
+    "ops",
+    [
+        lambda df: df.head(),
+        lambda df: df.iloc[1:100],
```

Comment:
Should these be moved to `test_iloc`?

Reply:
for connivence, I prefer putting them together.